tesseract/tesseract_planning/tesseract_process_managers/src/examples/freespace_manager_example.cpp

```diff
@@ -6,10 +6,10 @@ TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 TESSERACT_COMMON_IGNORE_WARNINGS_POP
 #include <tesseract/tesseract.h>
 #include <tesseract_command_language/command_language.h>
-#include <tesseract_motion_planners/trajopt/trajopt_motion_planner.h>
-#include <tesseract_motion_planners/trajopt/profile/trajopt_default_plan_profile.h>
-#include <tesseract_motion_planners/trajopt/profile/trajopt_default_composite_profile.h>
-#include <tesseract_motion_planners/core/utils.h>
+//#include <tesseract_motion_planners/trajopt/trajopt_motion_planner.h>
+//#include <tesseract_motion_planners/trajopt/profile/trajopt_default_plan_profile.h>
+//#include <tesseract_motion_planners/trajopt/profile/trajopt_default_composite_profile.h>
+//#include <tesseract_motion_planners/core/utils.h>
```

Comment:
Not sure if you want to leave these here commented out or remove them

Reply:
Removed.